### PR TITLE
RF 1.0.2.4 Documentation - Process to add a new environment for set up purpose 

### PR DESCRIPTION
**Pull request by EtricKombat-zz**
### To raise a new Pull Request, the following prerequisites need to be met. Please tick before proceeding:

- [x] a conversation was held in the appropriate media [hyperledgerkochi](https://github.com/orgs/hyperledgerkochi/projects/3) channel.
- [x] a Github [Issue](https://github.com/hyperledgerkochi/KOBman/issues) was opened for this feature / bug.
- [ ] test coverage was added .

____________________________________________________________________________________

**This addresses Issue** https://github.com/hyperledgerkochi/KOBman/issues/28

### CODE REVIEW

File: docs/user-guides/contributors_guide.md [hunk 1]
```diff
@@ -6,38 +6,57 @@ We love your input! We want to make contributing to this project as easy and tra
...
+- [Process for pull request](https://github.com/hyperledgerkochi/KOBman/blob/master/docs/user-guides/process-to-raise-a-pull-request.md)
+
+- [Process for issues](https://github.com/hyperledgerkochi/KOBman/blob/master/docs/user-guides/process-to-raise-an-issue.md)
+
+_______________________________________
 
 **Unfortunately we will close any Github Issues that have not followed this process of prior discussion.**
 
 ## Write bug reports with detail, background, and sample code
 [This is an example](https://github.com/asa1997/KOBman/issues/8) of a bug reported by a contributor .
```
Comment (asa1997) [inline docs/user-guides/contributors_guide.md:55]: @EtricKombat Please use the link for the issue section under hyperledgerkochi/KOBman

Comment (asa1997) [inline docs/user-guides/contributors_guide.md]: Please use the issue section for hyperledgerkochi/KOBman

File: docs/user-guides/contributors_guide.md [hunk 2]
```diff
@@ -51,21 +70,18 @@ After accepting a new feature or confirming that a bug was found, an Issue may b
 People *love* thorough bug reports.
```
Comment (asa1997) [inline docs/user-guides/contributors_guide.md:69]: @EtricKombat  Please use the issue section for hyperledgerkochi/KOBman for the sample issue

**Comment by asa1997** [comment]
@EtricKombat  Request you to address the comments

**Comment by EtricKombat-zz** [comment]
@asa1997 , the requested changes has been made.

**Comment by EtricKombat-zz** [comment]
@sumodgeorge  , the requested change has been made.

**Comment by EtricKombat-zz** [comment]
While the completion of the listed features, we will be able to define a rectified process .Will re-raise the same along with that.

RF 1.0.3.11 - Support for Java Development Environment
RF 1.0.3.10 - Support for Python Development Environment

**Pull request closed without merging**